samples/basic/blinky_pwm/boards/nucleo_wba55cg.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2022 STMicroelectronics
+ */
+
+
+/ {
+	pwmleds: pwmleds {
```

Comment:
yes, put node and alias in the board DTS

Reply:
What do you mean by complicated? It is a board feature, so it makes sense that it's defined at the board level for anyone to readily use. It's done that way already for literally dozens of Nucleo and Disco boards :)